server/controllers/students.ts

```diff
@@ -0,0 +1,36 @@
+import { NextFunction } from "express"
+import type { Request, Response } from "express";
+import { Class } from "../models/class";
+import { Students } from "../models/student";
+
+export const getStudentsRepeating = async (req: Request, res : Response, next : NextFunction) :Promise<void> =>{
```

Comment:
nex c'est pour des middleware
```suggestion
export const getStudentsRepeating = async (req: Request, res : Response) :Promise<void> =>{
```